File: collect/phpcover/testdata/sample/src/Greeter.php
<?php

declare(strict_types=1);

namespace Demo;

final class Greeter
{
    public static function greet(string $name, int $times): string
    {
        $total = Calculator::add($times, 0);

        return sprintf('Hello %s x%d', $name, $total);
    }
}

File: collect/dotnetcover/testdata/sample/Demo/Greeter.cs
namespace Demo;

public static class Greeter
{
    // Calls into Calculator so cross-file coverage is observable.
    public static string Greet(string name, int extra)
    {
        var count = Calculator.Add(1, extra);
        return string.Concat(System.Linq.Enumerable.Repeat($"hi {name}! ", count)).TrimEnd();
    }
}
